src/components/Checkbox/Checkbox.scss

```diff
@@ -147,7 +147,7 @@ $control-size: rem(16px);
   }
 }
 
-.error.globalTheming {
+.error.newDesignLanguage {
   .Backdrop {
     @include control-backdrop(error, $global-theming: true);
```

Comment:
I was going to mention this one should change too but this is removed as part of my PR